<launch>
  <node pkg="tf" type="static_transform_publisher" name="camera_to_robot" output="screen" 
    	args="0.01 0.125 0.12 0 0 -1.57079   /robot /usb_cam 10" />
 
  <node ns="usb_cam" pkg="image_proc" type="image_proc" name="image_proc"/>
 <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="camera_name" value="camera_logitec_c270"/>
    <param name="camera_info_url" type="string"
      value="file://$(find theatre_bot)/camera_calibration_files/camera_logitec_c270.yaml" />
<!--
	<param name="camera_name" value="lenovo_julian"/>
    <param name="camera_info_url" type="string"
      value="file://$(find theatre_bot)/camera_calibration_files/lenovo_julian.yaml" />
-->
  </node>	
 
 
 
	<arg name="marker_size" default="7.7" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/usb_cam/image_raw" />
	<arg name="cam_info_topic" default="/usb_cam/camera_info" />	
	<arg name="output_frame" default="usb_cam" />
 
	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
</launch>

